### ISSUE TITLE
Unable to start capture on multiple ports

### ISSUE THREAD

**Issue by Ravi-t** [pktgen/Pktgen-DPDK#35]
I have started pktgen with 2 ports as
pktgen l 0-7 -n 4 --proc-type auto --log-level 7 --socket-mem 2048 --file-prefix pg -w 0000:02:06.0 -w 0000:02:07.0 -- -m [1:2].1,[3:4].0 -l pktgen.log
but when  I give enable 0 capture or enable 1 capture, I get below error

![image](https://user-images.githubusercontent.com/27767393/93017436-0cc9ba00-f5c9-11ea-8793-98e3e0950c79.png)

My CPU layout is as below

![image](https://user-images.githubusercontent.com/27767393/93017606-55ce3e00-f5ca-11ea-882d-35d36548c589.png)

**Comment by pktgen** [comment]
Intel CPUs do not have more than 2 or 4 sockets and this must be AMD or ARM or something. Try removing the socket-mem it is not required much anymore.

**Comment by Ravi-t** [comment]
Dear Keith,

Its an intel CPU, basically I am running a Centos 7.x VM on Windows 10 host. I gave 8 cores to the VM, thats why it was showing 8 sockets, I reduced it to 4 cores and now I can see 4 sockets as below, I have also copied the /proc/cpuinfo output for first core showing that its intel CPU


![image](https://user-images.githubusercontent.com/27767393/93023072-be2e1700-f5ec-11ea-825c-7b276e9e8b41.png)

![image](https://user-images.githubusercontent.com/27767393/93023084-d2721400-f5ec-11ea-8095-8e63a9ccb4ee.png)

After reducing to 4 cores, I ran pktgen as below, and got below error when I tried "enable all capture".
Basically, its able to enable capture for port 0 but for port 1, I got the error.
Is there a way to allocate hugepages to individual sockets like socktet 0, socket 2 etc.? Right now, I just use normal command to allocate hugespages as below, 
**echo "2048" > /proc/sys/vm/nr_hugepages
mkdir /mnt/huge
mount -t hugetlbfs nodev /mnt/huge**

pktgen l 0-3 -n 4 --proc-type auto --log-level 7 --file-prefix pg -w 0000:02:06.0 -w 0000:02:07.0 -- -m [1:2].1,[3].0 -l pktgen.log 

![image](https://user-images.githubusercontent.com/27767393/93023141-2f6dca00-f5ed-11ea-8452-dc5ed9fb6608.png)
